### PR TITLE
Update filebeat to new Run structure to simplify main file

### PR DESCRIPTION
**Pull request by ruflin**

### CODE REVIEW

File: etc/filebeat.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ filebeat:
       #backoff: 1s
 
       # Max backoff defines what the maximum backoff time is. After having backed off multiple times
-      # from checking the files, the waiting time will never exceed max_backoff independent of the
+      # from checking the files, the waiting time will never exceed max_backoff idenependent of the
```
Comment (ruflin) [inline etc/filebeat.yml]: This should be corrected automatically again as soon as we merge in 1.0.0 changes

File: main.go [hunk 2]
```diff
@@ -23,28 +18,5 @@ var GlobalBeat *beat.Beat
...
-	if err != nil {
-		logp.Critical("Config error: %v", err)
-		os.Exit(1)
-	}
-
-	// Run beat. This calls first beater.Setup,
-	// then beater.Run and beater.Cleanup in the end
-	b.Run()
-
+	beat.Run(Name, Version, &filebeat.Filebeat{})
```
Comment (urso) [inline main.go]: I'd like a filebeat.New() here. But just minor detail

Comment (ruflin) [inline main.go]: @urso I like that suggestion. Added.

**Comment by monicasarbu** [comment]
LGTM

**Pull request merged**